bundles/org.openhab.binding.telegram/README.md

```diff
@@ -52,6 +52,29 @@ Please note that the things cannot be used to send messages. In order to send a
 | `chatIds`               |         | Yes      | Comma-separated list of chat ids                                                             |
 | `botToken`              |         | Yes      | authentication token                                                                         |
 | `parseMode`             |  None   | No       | Support for formatted messages, values: Markdown or HTML.                                    |
+| `proxyHost`             |  None   | No       | Proxy host for telegram binding, it doesn't affect entire openhab.                           |
+| `proxyPort`             |  None   | No       | Proxy port for telegram binding.                                                             |
+| `proxyType`             |  SOCKS5 | No       | Type of proxy server for telegram binding. Default: SOCKS5                                   |
+
+
+telegram.thing (no proxy):
+```
+Thing telegram:telegramBot:Telegram_Bot [ chatIds="< ID >", botToken="< TOKEN >" ]
+```
+
+
+telegram.thing (SOCKS5 proxy server is used): 
+```
+Thing telegram:telegramBot:Telegram_Bot [ chatIds="< ID >", botToken="< TOKEN >", proxyHost="localhost", proxyPort="9050", proxyType="SOCKS5" ]
```

Comment:
For the id and the token you used some kind of placeholder like "< ID >", but for the proxyHost and the proxyPort you used a specific example and not "something like "< HOST>" and "< PORT >". Can we keep that consistent?

Reply:
> For the id and the token you used some kind of placeholder like "< ID >", but for the proxyHost and the proxyPort you used a specific example and not "something like "< HOST>" and "< PORT >". Can we keep that consistent?

Sure, I'll doo

Reply:
> For the id and the token you used some kind of placeholder like "< ID >", but for the proxyHost and the proxyPort you used a specific example and not "something like "< HOST>" and "< PORT >". Can we keep that consistent?

fixed